npm/react/cypress/component/advanced/set-timeout-example/loading-indicator.cy.jsx

```diff
@@ -73,7 +73,9 @@ describe('LoadingIndicator', () => {
       )
 
       cy.tick(2010)
-      unmount()
+
+      // mount something else to trigger unmount
+      mount(<div>Test Component</div>)
```

Comment:
Should we use the suggested alternative:
`cy.then(() => ReactDom.unmountComponentAtNode(getContainerEl()))`